### ISSUE TITLE
Is this plugin the same as rhubarb's GBrowse?

### ISSUE THREAD

**Issue by TamaMcGlinn** [tyru/open-browser-github.vim#41]
With tpope's [rhubarb](https://github.com/tpope/vim-rhubarb) installed, it seems I get the same functionality with these two mappings:

```
nnoremap <leader>gx :GBrowse<CR>
vnoremap <leader>gx :GBrowse<CR>
```

Is there any difference?

**Comment by TamaMcGlinn** [comment]
For instance, I generated [this link](https://github.com/TamaMcGlinn/vimrc/blob/ef9b86c3d3011ea24839726dff74819be71becd5/git.vim#L165-L166) by selecting those lines of my vimrc and pressing `<leader>gx`.

**Comment by tyru** [comment]
`:GBrowse` seems has only [`:OpenGithubFile`](https://github.com/tyru/open-browser-github.vim#opengithubfile) feature.
This plugin provides many accesses to the pages on github.

By the way, I found that `:OpenGithubCommit {commit hash}` command is missing in README.
See full commands for [doc](https://github.com/tyru/open-browser-github.vim/blob/ac7c034e300f36d591ef234dcd5eb5cd5c07c74f/doc/openbrowser-github.txt#L117-L214)

**Comment by TamaMcGlinn** [comment]
No, rhubarb also allows me to open a commit with the same GBrowse command. From a fugitive buffer (e.g. from `:Git blame` and then enter on some commit hash) which is showing a commit, GBrowse will open the browser on that commit.

It also allows me to name a commit; if I am looking at some file which belongs to a git repo, I can do e.g. `:GBrowse master` to open master, or `:GBrowse vimrc.vim` to browse that file (on the master branch).

It also works [for gitlab](https://github.com/shumphrey/fugitive-gitlab.vim), or even a [custom gerrit server](https://github.com/TamaMcGlinn/vim-gerritbrowse), by installing GBrowse handlers.
